### ISSUE TITLE
Site down for download

### ISSUE THREAD

**Issue by Cyberdr3am** [moudey/Shell#457]
Hello,

I encountered an issue while trying to access the Shell Program website. It seems that the website is returning an error 503. This error typically indicates that the server is temporarily unavailable due to maintenance or overloading.

Could you please look into this issue and provide an update on when the website will be accessible again? I rely on the Shell Program website for important information, and any delay in accessing it could affect my work.

Thank you for your attention to this matter.

Best regards

**Comment by keiwsh** [comment]
Also needed to install and website was down ->  therefore went on archive.org and downloaded the package.
https://web.archive.org/web/20240405193715/https://nilesoft.org/download

**Comment by BrainB0ne** [comment]
I also noticed that the nilesoft website is down at this moment. Thank you for the webarchive link to get the setup file.

**Comment by iMAboud** [comment]
Yeah, this is the second time I encountered the website being down and used your archive link.
I'll have to pin this page for later

**Comment by Dhyfer1** [comment]
@moudey At this moment I also cannot access the website to download Shell. Why don't you publish the installer in your repository like everyone else does? 🤨

**Comment by iMAboud** [comment]
@Dhyfer1 Since we brought this up you might wanna check my latest script, it sends files between devices and can actually installs Nilesoft for you without relying on the website 😁

**Comment by Dhyfer1** [comment]
> @Dhyfer1 Since we brought this up you might wanna check my latest script, it sends files between devices and can actually installs Nilesoft for you without relying on the website 😁

Thx. Looks interesting 🤔 I'll give it a try

**Comment by SleepDaemon** [comment]
1.9.15 is what i downloaded from Stable, but the context menu is all white instead of being black

**Comment by SleepDaemon** [comment]
> 1.9.15 is what i downloaded from Stable, but the context menu is all white instead of being black

Update: From the web archive, download the canary version which follows which mode you're in (light or dark), stable will download a newer version that is broken.

**Comment by SleepDaemon** [comment]
Update: Canary version from the webarchive (1.9.14) has this issue, anyone have 1.9.4 it was last working one for me

![image](https://github.com/moudey/Shell/assets/88533953/8b6f37ac-0e91-4524-9686-4ab47c40cd48)

**Comment by SleepDaemon** [comment]
> > ![image](https://private-user-images.githubusercontent.com/88533953/328522690-8b6f37ac-0e91-4524-9686-4ab47c40cd48.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTUwODU4NTMsIm5iZiI6MTcxNTA4NTU1MywicGF0aCI6Ii84ODUzMzk1My8zMjg1MjI2OTAtOGI2ZjM3YWMtMGU5MS00NTI0LTk2ODYtNGFiNDdjNDBjZDQ4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MDclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTA3VDEyMzkxM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTAxZmQ2ZDRiYzZiZDY2ZmI5NjQwYTY0MTZjYjYxYmM2Mzg0ZmVhNjZkNzg1NDlhZDVjODJkOWQxNThkMDFlN2EmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.B8OUHhrRWeaiowvi_DD9d__ibgBZAmbnSorDOYMsOP8)
> 
> Does that colored border in the screenshot come from sharex?

yes

**Comment by SleepDaemon** [comment]
Update: Found download for 1.9.6 canary which works for me with no issues

Link: https://web.archive.org/web/20231208150639/nilesoft.org/download
